Add description for status and metrics

axelsiebert opened the pull request:
Adds documentation for the Spring-Boot-Actuator endpoints and what their purpose will be.

I added a note on the first version of EHRbase supporting this feature which might be release 0.15.0. as an idea how we can also inform the user which version to use at least to get this feature.

axelsiebert committed:
Add description for status and metrics

stefanspiska commented on source/03_development/08_status_and_metrics/00_security/index.rst:
```diff
@@ -0,0 +1,19 @@
+EHRbase integrated security solutions. If you are using at least the *Basic Auth* method to secure
+incoming requests the endpoints are only available for users with the *Admin*-Role.
+
+If you want to access specific metric and health endpoint from a statistics and management service
+ensure not sending the passwords in clear text. For Basic Auth encode them into a Base64 string
+as described here: :doc:`Admin API security<../../07_admin/00_security/index>`.
+
```
I don't unterstand way this is more secure since you can easy decode base64

stefanspiska replied:
I also don't see any comments on that in  ../../07_admin/00_security/index

axelsiebert replied:
I changed the link to the proper documentation in the EHRbase repository.

Also added a note about unsafe usage of Base64 Encoded string and recommend a mor secure Auth Method

stefanspiska reviewed (changes requested):
see comments

axelsiebert committed:
PR feedback

:pencil: Add note on insecure usage of Base64 String
:pencil: Correct link to proper security documentation.

axelsiebert committed:
Correct images path

:pencil: Change images to be local to page
:pencil: Add Coker env instructions

axelsiebert committed:
Add env description

:pencil: Add description on config from environment variables
:pencil: Change security note warn about insecure communication
:package: Remove chapter numbers from folders
